jsx/App/Stories/Story/Display/TextFormatButton.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import TierSelectionWindow from "./TierSelectionWindow.jsx";
+
+/* 
+  A text format button that renders a window for tier selection on clicked. 
+  This tier selection window then leads to a new block displaying the result of LaTeX format conversion.
+*/
+export class TextFormatButton extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            buttonClicked : false
+        }
+        this.handleClick = this.handleClick.bind(this);
+      }
+
+    /* Updates the flag when button is clicked so that the tier selection component gets rendered. */
+    handleClick(e) {
+        e.preventDefault();
+
+        this.setState({
+            buttonClicked : true
+        });
+
+    }
+
+    componentDidMount() {
+        this.setState({ 
+            sentence : this.props.sentence,
+            metadata : this.props.metadata
+        });
+    }
+
+    render() {
+        return (
+            <div class="textFormatSection">
+                <button class="textFormatButton" onClick={this.handleClick}>
+                    Format
```

Comment:
Let's use a `<TranslatableText>` component for the word "Format", like this: `<TranslatableText dictionary={formatButtonText} />` and then define formatButtonText in `jsx/App/locale/LocaleConstants.jsx`. For the French translation, Google Translate is fine.

Reply:
Same goes for the other pieces of user-visible text: "Please select the tier that corresponds to each section in the LaTeX formating.", "original sentence", "Confirm", "Story title:", etc. Don't translate the tier names though, since we don't know what they are in advance.

If you're running short on time, we can merge the PR without TranslatableText and I can add it later, since it's a pretty mechanical process. :)

Reply:
I updated all the button texts with a locale constant. Didn't have time to change the other texts on the tier selection form..

Reply:
Thanks! I'll take care of the rest of these.